File: LeetCode/1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        unordered_map<char, int> mp;
        for (int i = 0; i < s.length(); i++) {
            mp[s[i]]--;
            mp[t[i]]++;
        }
        int count = 0;
        for (int i = 0; i < t.length(); i++) {
            if (mp[t[i]] > 0) {
                count += mp[t[i]];
                mp[t[i]] = 0;
            }
        }
        return count;
     }
};

File: LeetCode/1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        mp = defaultdict(int)
        
        for charS, charT in zip(s,t):
            mp[charS] -= 1
            mp[charT] += 1
            
        count = 0
        
        for charT in t:
            if mp[charT] > 0:
                count += mp[charT]
                mp[charT] = 0
                
        return count